app/retail/emails.py

```diff
@@ -67,6 +68,102 @@ def premailer_transform(html):
     return p.transform()
 
 
+def render_new_grant_email(grant):
+    params = {'grant': grant}
+    response_html = premailer_transform(render_to_string("emails/grants/new_grant.html", params))
+    response_txt = render_to_string("emails/grants/new_grant.txt", params)
+    subject = "Your Gitcoin Grant"
```

Comment:
IMHO its worth putting the grant title in each of these subject lines (people may have multiple grants)